### PR TITLE
Travis update

### PR DESCRIPTION
**Pull request by roboMoose**
updates to the configuration:
Semi-colons are back and the server files are now tested. Prettier is in-place and will ignore the test folder.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,15 +11,20 @@
...
   "repository": {
     "type": "git",
     "url": "git+https://github.com/1904-cpu-projects/errant-outfitters.git"
   },
   "keywords": [],
+  "prettier": {
+    "semi": true,
+    "singleQuote": true,
+    "trailingComma": "all"
+  },
```
Comment (nicknapoli82) [inline package.json:27]: By adding this in. Your defining the same thing as you have in the "prettier" script. I don't think it really matters actually, but I think having this defined negates the need for ```--single-quote```.

File: package.json [hunk 2]
```diff
@@ -11,15 +11,20 @@
     "start:dev": "npm run webpack:dev & nodemon server/index.js --ignore public --ignore src",
     "test": "node server/db/seed.js & jest",
     "test:watch": "jest --watchAll",
-    "lint": "eslint src",
-    "lint:fix": "eslint --fix src",
-    "prettier": "prettier --write --no-semi --single-quote \"src/**/*.js\""
+    "lint": "eslint src server",
+    "lint:fix": "eslint --fix src server",
+    "prettier": "prettier --write --single-quote \"src/**/*.js\" \"server/**/*.js\" "
```
Comment (nicknapoli82) [inline package.json:16]: Did you intend to add a --ignore in this for our test files? Or just let them be parsed by prettier also?

Comment (roboMoose) [reply package.json:16]: Oh - i meant to take that out completely. it should just reference the rules defined lower in the file

Comment (nicknapoli82) [reply package.json:16]: I think we just fix now before we say this is good to go then.

Comment (roboMoose) [reply package.json:16]: give me a sec to remove from the file

Comment (roboMoose) [reply package.json:16]: sorry i read your comment too quickly. I made a .ignoreprettier file which houses the tests. I figure this was easier especially if we decide to include other file or folder types

Comment (nicknapoli82) [reply package.json:16]: Oh. I missed that one. It was literally right at the top of the files changed.

**Review by nicknapoli82 — APPROVED** [review]

**Pull request merged**